src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java

```diff
@@ -1479,6 +1480,181 @@ public void testResultSetErrors() throws Exception {
             }
         }
 
+        /**
+         * Tests result of math operation in prepared statement using subtraction
+         * 
+         * @throws SQLException
+         *         when an error occurs
+         */
+        @Test
+        public void testMathBigDecimalSubtraction() throws SQLException {
+            try (Connection con = getConnection(); Statement stmt = con.createStatement()) {
+                stmt.executeUpdate("create table " + AbstractSQLGenerator.escapeIdentifier(tableName)
```

Comment:
instead of calling AbstractSQLGenerator.escapeIdentifier everywhere can we just do this once where tableName is defined

Reply:
I don't see why not. Every instance of `tableName` uses it in the context of `AbstractSQLGeneratior.escapeIdentifier(tableName)`, so this would help for clarity sake. I'll try it out.